### DISCUSSION TITLE
AM mode adjustments please

### DISCUSSION THREAD

**Discussion by N8SDR007** [ExpertSDR3/ExpertSDR3-BUG-TRACKER#60] [Ideas]
These all apply to the AM mode

1)	The 5-minute total TX timer- While I understand why you guys have done this, I think from a safety aspect of keeping the rig cool
In AM mode this is a real PITA, since many AM users usually go on 5-10 minute TX sessions at a single time, IN AM mode this really should not cause too much an issue in my view since the carrier level is low( even at 25 watts) and if decent air around the 2DX or MB1 since both have  fan , can this TX timer be adjusted in future- I personally know if 2 users that were extremely happy with the 2DX , but have moved to ANAN because of ONLY THE TX TIMMER ISSUE!

2)	Mode- AM (Standard) and AM synchronous demodulation- Yes there are certainly many benefits to Synchronous AM however I have and others experienced issues when ONLY this option is available and when other CLOSE Strong AM stations are on Frequency within 4-5KC, the lock has issues, and the stronger station usually wins, or the audio is very badly mixed because of two competing stations in close frequency range. Like in previous release of EESDRE before V3 can V3 have option of NON and Synchronous AM- be able to switch to NON in a crowded passband with strong signals is actually extremely helpful! (Here in US, there is still a large number of AM users with some very NICE old IRON broadcast AM TX conversions, and they have big signals and the allocations of frequency spacing is absurd for that mode here in US we get crammed into a very small portion of the bands- not always a good outcome for an AM synchronous mode only at that point things get bad on RX!

3)	Ability to control Carrier offset- Okay I know in item #1 I mentioned the lower carrier levels keeping the thermals inline,  I’m not asking for 100% control of carrier level but perhaps ability to adjust lower or possibly slightly higher than the standard ¼ percent of total PEAK power modulated – so say you have a peak power around 60 watts on peak normally that carrier level is 25% of the peak so 15 watts carrier have the ability to slightly raise the carrier to say a max of 30%-35% percent of carrier can make a big difference on using an Amplifier- Case in point my ACOM 1000 like a minimum drive level of around 25-30 watts MIN to operate in a liner class state- but doesn’t like to have more then 75-80 watts PEAK modulated or carrier into it.  The LOWE carrier level of 15-watts make it a hard thing to keep linear on the out of the AMP as now I have a carrier of about 80-10o watts and a PEAK Modulated AM product around 800+ I’m no longer in that ¼ ratio – by allowing a user to increase the carrier level to 18- 22 watts this makes it more manageable and now the amp is inline to a more linear output state and the CLASS of the amplifier is not upset because of a LOW drive to High peak ratio.

Thanks

Rick N8SDR

**Comment by Kb2uka** [comment]
I would like to see these features added as well.
